lua/nvim-tree/utils.lua

```diff
@@ -42,6 +44,21 @@ function M.path_split(path)
   return path:gmatch("[^" .. path_separator .. "]+" .. path_separator .. "?")
 end
 
+function M.norm_path(path)
+  if M.is_windows then
```

Comment:
I left a comment in utils.lua `-- false for WSL` however this (unchanged) code indicates that comment is incorrect.

We're not changing anything so I'll remove it.